lib/node_modules/@stdlib/blas/base/ssyr/README.md

```diff
@@ -183,21 +183,62 @@ console.log( A );
 ### Usage
 
 ```c
-TODO
+#include "stdlib/blas/base/ssyr.h"
+#include "stdlib/blas/base/shared.h"
```

Comment:
Move this include to the examples, rather than as part of the `ssyr` include code block.